lnd_test.go

```diff
@@ -3115,6 +3412,10 @@ var testsCases = []*testCase{
 		name: "revoked uncooperative close retribution",
 		test: testRevokedCloseRetribution,
 	},
+	{
+		name: "revoked uncooperative close retribution post breach conf",
+		test: testRevokedCloseRetributionPostBreachConf,
```

Comment:
I would like to roll these together, though initially this was done to ensure that we were working with a clean slate, and that i wasn't introducing bugs into the test case itself. After talking w/roasbeef, final verdict was to refactor in 2nd-layer htlc PR, along with more robust refactoring of the unit tests.